### PR TITLE
from type isn't an optional part

### PR DESCRIPTION
**Pull request by siggemannen**
PR for issue #9463 , from type isn't optional

### CODE REVIEW

**Comment by Court72** [comment]
@markingmyname

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/t-sql/statements/create-type-transact-sql.md [hunk 1]
```diff
@@ -42,11 +42,10 @@ dev_langs:
 -- User-defined Data Type syntax    
 CREATE TYPE [ schema_name. ] type_name  
 {   
-    [
+
```
Comment (rwestMSFT) [inline docs/t-sql/statements/create-type-transact-sql.md]:
```suggestion
```

**Comment by rwestMSFT** [comment]
We're in a content freeze this week, but we should be able to get this merged in the next few days. Thanks again for your contribution, @siggemannen.

**Comment by markingmyname** [comment]
#sign-off

**Comment by Jak-MS** [comment]
Due to the upcoming Ignite conference, all merging and publishing is on hold until the morning of 11/15.

This PR is ready for merge on 11/15.

#label: "event-triaged-to-10AM"
#label: "missed-content-complete"
#label: "aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Pull request merged**